Source/GameBaseFramework/Private/Animation/GBFAnimNotifyState_InputBuffer.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Animation/GBFAnimNotifyState_InputBuffer.h"
+
+#include "GBFLog.h"
+
+#include <Components/SkeletalMeshComponent.h>
+
+void UGBFAnimNotifyState_InputBuffer::NotifyBegin( USkeletalMeshComponent * mesh_component, UAnimSequenceBase * animation, float total_duration, const FAnimNotifyEventReference & event_reference )
+{
+    Super::NotifyBegin( mesh_component, animation, total_duration, event_reference );
+
+    if ( auto * aibc = GetAbilityInputBufferComponent( mesh_component ) )
```

Comment:
I would not use abbreviations like this.
asc, pc, ps etc. I can understand since they are base types in UE.
Since this is a custom type, it's hard to understand what type it is from just the name

Reply:
Done